presto-atop/src/main/java/com/facebook/presto/atop/AtopSplit.java

```diff
@@ -56,6 +59,17 @@ public HostAddress getHost()
     }
 
     @JsonProperty
+    public long getInstant()
```

Comment:
How about switching this class to use `ZonedDateTime`? That will serialize properly since it's part of the JDK (and Presto guarantees to include the proper Jackson modules for it).